### ISSUE TITLE
Ummm, what happened???

### ISSUE THREAD

**Issue by DCCXVIII** [moody/Dejunk#122]
The addon updated and poof. All my custom filters gone. All my inclusions and exclusions gone. This addon was the only junk addon that exists that allowed for decent filtering and ease of use. Now its been reduced to some stripped down to the barebones and lost the vast majority of its features. Why???

**Comment by WhyDaRumGone** [comment]
All mines gone. If you go to forge you can get the old version back BUT I wasn't able to figure out a way to get back the saved variables.

**Comment by ChesterBay** [comment]
developer decided to rebuild the entire addon instead of doing a new addon and ruined the old one :)

**Comment by zalidthebur** [comment]
Hopefully this will get fixed. Until then I'll stick with the old version ;-)

**Comment by Rykkan** [comment]
I will admit I love the new interface. But I will stick with the last version because there is no sell filters by ilvl or anything.

**Comment by zalidthebur** [comment]
I took the plunge and went with the new one. It means rebuilding my include list. But I do like that the items in the sell window now show tooltips.

**Comment by DCCXVIII** [comment]
> I took the plunge and went with the new one. It means rebuilding my include list. But I do like that the items in the sell window now show tooltips.

I see. Well I guess we'll have to adapt! Thanks for responding and continuing development.

Would it be possible to put the ilvl range back in? Because right now it's selling stuff I don't want it to since we no longer have control over what it deems as lower ilvl. E.g. if you have items in your bags that are low ilvl due to bad rng but you need to keep for the alt specs. I know I could add them to the exception list but then that list would get huge over time.

Also can we get our destroy list back? I know it can't destroy multiple things anymore due to API changes but in the old add-on I could spam right click the minimap icon to rapidly delete items. It was great.

**Comment by moody** [comment]
Saved variables from the 9.1.2 version get overwritten when updating to the new version of Dejunk because of a sloppy mistake I made. When coding the new version, I modified the names of the saved variable tables (as defined in the .toc files), because I believed that this would leave the old saved variable tables intact (in the event that anyone wanted to switch back to 9.1.2). However, by only modifying the names I effectively removed the names of the old saved variables in the .toc files, so the game client ended up erasing them entirely. Again, really dumb mistake on my part, and I don't blame anyone for being pissed about it.

**Comment by glassleo** [comment]
Nevermind years of settings just disappeared when I updated this now....

But I'm so confused by the new inferface. Can you no longer have both global *and* profile based inclusions/exclusions? This is a huge downgrade.

There's also no mass import?

**Comment by AmidaShamida** [comment]
What I miss is the auto sell via an item level threshold. The below average item level is too high. I liked setting it about 40 ilvls lower to auto sell everything from old content. Still the best auto selling addon around.

**Comment by WhyDaRumGone** [comment]
I agree

**Comment by moody** [comment]
> What I miss is the auto sell via an item level threshold. The below average item level is too high. I liked setting it about 40 ilvls lower to auto sell everything from old content. Still the best auto selling addon around.

I'm going to work on adding this back soon. The current "Include Below Average Item Level" setting is a failed idea.

**Comment by WhyDaRumGone** [comment]
> Nevermind years of settings just disappeared when I updated this now....
> 
> But I'm so confused by the new inferface. Can you no longer have both global _and_ profile based inclusions/exclusions? This is a huge downgrade.

I noticed this too. You can click the "Character specific" settings BUT it WONT use the global settings at all. It's an all or nothing sort of deal now.

**Comment by glassleo** [comment]
> > Can you no longer have both global _and_ profile based inclusions/exclusions?
> 
> I noticed this too. You can click the "Character specific" settings BUT it WONT use the global settings at all. It's an all or nothing sort of deal now, and you will have to duplicate all your global stuff to your individual characters settings

Yeah... I really hope this can be changed... I just miss the old version so much right now, it's been difficult to wrap my head around these changes and trying to rebuild my destroy lists.

**Comment by tomlarose** [comment]
New addon should be called Junk rather than Dejunk.  What version was the old working one so I can download that instead of this hot mess?
